requirements.txt

```diff
@@ -10,6 +10,8 @@ tensorflow>=2.2
 tfa-nightly==0.12.0.dev20200820045606
 PyYAML>=3.12
 absl-py
+kaggle
+slugify
```

Comment:
Do we need `slugify`?

Reply:
Yes kaggle has a dependency on that

Reply:
Looks like they already pull in `python-slugify` in their requirements: https://github.com/Kaggle/kaggle-api/blob/master/setup.py#L39

Are you sure we need both?  That's a bug on their end if so.

Reply:
I thought I saw one pulling in the other, I think you're right, will remove and rebuild

Reply:
Removed